prism/src/main/java/org/apache/falcon/resource/AbstractExtensionManager.java

```diff
@@ -177,4 +177,22 @@ public static String getJobNameFromTag(String tags) {
         }
         return tags.substring(nameStart, nameEnd);
     }
+
+    public String disableExtension(String extensionName, String currentUser) {
+        validateExtensionName(extensionName);
+        try {
+            return ExtensionStore.get().disableExtension(extensionName, currentUser);
```

Comment:
You can remove enable and disableExtension and directly call updateExtension with enable or disable as parameter